GVRf/Framework/framework/src/main/java/org/gearvrf/GVRJassimpAdapter.java

```diff
@@ -743,13 +796,21 @@ private void loadTexture(GVRAssetLoader.AssetRequest assetRequest,
         }
         if (texIndex > 0)
         {
+            if (usingPBR)
```

Comment:
Why does this have to special-cased for GVRPBRShader?

Reply:
Because assimp puts multiple textures of the same kind into the file and they are redundant. It causes our code to make layered textures (because that is how they are normally expressed by assimp). The support for GLTF in assimp is in development. We should rewrite JassimpAdapter.java and jassimp.cpp in C++ and have it create GearVRF objects directly. Then we could filter out the crud before it gets to the asset loader instead of make generic Java objects from it and filtering out the crud after.